### PR TITLE
Correct Standard other lines

### PR DESCRIPTION
**Pull request by rafaelstz**
This PR I standardized paragraphs. Before they were different.

### CODE REVIEW

**Comment by sedge** [comment]
@rafaelstz Thanks for the contribution! Can you update the pull request description to describe what you're fixing in this patch?

**Comment by rafaelstz** [comment]
Done @sedge :+1: :smile:

File: locale/en_US/css-property-docs.json [hunk 1]
```diff
@@ -1,173 +1,173 @@
...
+    "animation-delay": "The <code>animation-delay</code> CSS property specifies when the animation should start. This lets the animation sequence begin some time after it's applied to an element.",
+    "animation-direction": "The <code>animation-direction</code> CSS property indicates whether the animation should play in reverse on alternate cycles.",
+    "animation-duration": "The <code>animation-duration</code> CSS property specifies the length of time that an animation should take to complete one cycle.",
+    "animation-iteration-count": "The <code>animation-iteration-count</code> CSS property defines the number of times an animation cycle should be played before stopping.",
+    "animation-name": "The <code>animation-name</code> CSS property specifies a list of animations that should be applied to the selected element. Each name indicates a <code><a href=\"https://developer.mozilla.org/CSS/@keyframes\">@keyframes</a></code> at-rule that defines the property values for the animation sequence.",
```
Comment (sedge) [inline locale/en_US/css-property-docs.json]: Why have you removed the link to the CSS section of MDN?

**Comment by rafaelstz** [comment]
The first link is the main, the other links below are just passing the property animation parameters.

**Comment by sedge** [comment]
@rafaelstz Were you trying to fix [thimble.webmaker.org](http://thimble.webmaker.org)? If so, you need to re-write this patch using the `master` branch instead of `bramble`. 

The `bramble` branch contains the new form of thimble, which you can use at [thimble.mozilla.org](http://thimble.mozilla.org). These files are artifacts that should be deleted on this branch, since we aren't using them in the new thimble. We just haven't had the time :neutral_face:

**Comment by sedge** [comment]
@rafaelstz If you're looking for ways to help, our new thimble editor needs a lot of work! We'll be starting localization in the next couple of weeks. We're always looking for new help :smile: 

cc: @hannahkane

**Comment by rafaelstz** [comment]
I am really motivated to help in this project. How the team communicates?

**Comment by hannahkane** [comment]
Hey @rafaelstz - thanks for your interest and your contributions! You can join us in the #thimble channel of irc.mozilla.org.

**Comment by rafaelstz** [comment]
Thanks! @hannahkane @sedge

**Pull request closed without merging**